operations/host_spawn.go

```diff
@@ -438,6 +438,56 @@ func hostDetach() cli.Command {
 	}
 }
 
+func hostListVolume() cli.Command {
+	return cli.Command{
+		Name:  "list",
+		Usage: "list volumes for user",
+		Action: func(c *cli.Context) error {
+			confPath := c.Parent().Parent().String(confFlagName)
+			ctx, cancel := context.WithCancel(context.Background())
+			defer cancel()
+
+			conf, err := NewClientSettings(confPath)
+			if err != nil {
+				return errors.Wrap(err, "problem loading configuration")
+			}
+			client := conf.getRestCommunicator(ctx)
+			defer client.Close()
+
+			volumes, err := client.GetVolumesByUser(ctx)
```

Comment:
do we remove volumes when they no longer exist in AWS?

Reply:
We remove volumes from the DB when users delete them. Theoretically if you manually went into AWS and deleted a volume then it would still be in the database.

Reply:
Got it.
If a user spawns a host that comes up with EBS volumes will those volumes be included in this list?

Reply:
No, only volumes created by the user will be here.

Reply:
Wouldn't we want them listed as well? 
Let's say you start an instance with an EBS volume and you decide you want a larger instance. You could migrate all your stuff to the new machine by moving the volume.
If the only volumes you see are the ones you created manually then you'd only be able move volumes you created manually.

Reply:
Resolved in person -- only user created volumes will be listed.